proto/warden/async/v1beta1/future.proto

```
@@ -0,0 +1,55 @@
+  // Vote type.
+  FutureVoteType vote = 3;
+}
+
+enum FutureVoteType {
+  // Unspecified vote type.
+  VOTE_TYPE_UNSPECIFIED = 0;
+  // Vote to approve the result of the Future.
+  VOTE_TYPE_VERIFIED = 1;
+  // Vote to reject the result of the Future.
+  VOTE_TYPE_REJECTED = 2;
+}
```

buf (1.47.2)

Enum "FutureVoteType" should have a non-empty comment for documentation.

(COMMENT_ENUM)